engine/apps/schedules/ical_utils.py

```diff
@@ -90,14 +93,9 @@ def users_in_ical(
         (Q(username__in=usernames_from_ical) | Q(email__lower__in=emails_from_ical))
     ).distinct()
```

Comment:
Wondering if we should do a `select_related` for organization here, since we will be accessing `u.organization` for each user when checking `user_has_permission` below.

Reply:
good catch 👍 addressed in https://github.com/grafana/oncall/pull/5149/commits/6c9a29dc746a8ccea62ce4de3223383a3cd84905